Staging

wli-pro opened the pull request:
### Context

<!-- Why are these changes needed? Please use auto-close keyword to link to an existing issue, if any -->

Try sync staging to dev branch

### Labels

Please apply following PR-related labels when appropriate:
- `C0-breaking`: if your change could break the existing client, e.g. API change, critical logic change 
- `C1-noteworthy`: if your change is non-breaking, but is still worth noticing for the client, e.g. reference code improvement

### How (Optional)

<!-- How were these changes implemented? -->

### Testing Evidences

Please attach any relevent evidences if applicable

wli-pro committed:
feat: #1697 added support for new discord username, while old username would still work (#1876)

Co-authored-by: higherordertech <higherordertech>

Kailai-Wang committed:
Try to skip timeslot check for single worker

Kailai-Wang committed:
add DI support for A13

Kailai-Wang committed:
fix logic error

Kailai-Wang commented:
I don't get this PR...

The majority of changes should be in `dev` already, except for the suppression of time slot check

BillyWooo committed:
change back to old graphql url.
This is not valuable for committing to dev branch. Because it's still the old interface, which will be replaced in new Achainable API. Here still use it is because the transfer is not yet completely finished. Once transfer is done. It will be removed/overwrite.

BillyWooo reviewed (commented):
Hi @higherordertech , I think it should be the other way around first. But time is not yet appropriate yet.
